public/javascripts/table.jsx

```diff
@@ -0,0 +1,82 @@
+var DeviceRow = React.createClass({
+  lastSeen : function(timestamp) {
+    var second = 1000;
```

Comment:
I dunno if these should be seen as constants... They're more like singleton's I think. Also a file called constants would get pretty messy and have all sorts of constants. IMO having these in a table class is fine but more input is welcome